trouble-shooting.md

```diff
@@ -6,11 +6,11 @@ category: advanced
 
 # TiDB Cluster Troubleshooting Guide
 
-You can use this guide to help you diagnose and solve basic problems while using TiDB. If your problem is not resolved, please collect the following information and [create an issue](https://github.com/pingcap/tidb/issues/new):
+You can use this guide to help you diagnose and solve basic problems while using TiDB. If your problem is not resolved, please collect the following information and [create an issue](https://github.com/pingcap/tidb/issues/new?template=bug-report.md):
```

Comment:
I suggest using the following link because it offers multiple choices.

```suggestion
You can use this guide to help you diagnose and solve basic problems while using TiDB. If your problem is not resolved, please collect the following information and [create an issue](https://github.com/pingcap/tidb/issues/new/choose):
```